openstack/keystone/files/wsgi-keystone.conf

```diff
@@ -1,8 +1,9 @@
+{% from "openstack/worker_pool.jinja" import worker_count %}
 Listen 5000
 Listen 35357
 
```

Comment:
you verify all these other special chars pass through the jinja parser properly?